mypy.ini

```diff
@@ -1,8 +1,10 @@
 [mypy]
 python_version = 3.8
 mypy_path = mypy_stubs
-# local-dev is excluded until we decide what to do with __init__.py in those packages.
-exclude = local-dev/
+exclude = (?x)(
+    ^buildman/test/test_buildman\.py$ |
+    ^buildman/buildman_pb/buildman_pb2\.py$ |
+    ^endpoints/api/test/test_security\.py$ )
```

Comment:
Lots of wildcard imports confuse mypy:
https://github.com/quay/quay/blob/922a82a3d945002dd2d259d2d7950614249ef382/endpoints/api/test/test_security.py#L12-L38
```
endpoints/api/test/test_security.py:16: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:17: error: Name "datetime" already defined (by an import)
endpoints/api/test/test_security.py:20: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:22: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:23: error: Name "datetime" already defined (by an import)
endpoints/api/test/test_security.py:23: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:24: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:26: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:27: error: Incompatible import of "Repository" (imported name has type "Type[data.database.Repository]", local name has type "Type[endpoints.api.repository.Repository]")
endpoints/api/test/test_security.py:29: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:30: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:31: error: Name "model" already defined (by an import)
endpoints/api/test/test_security.py:32: error: Name "datetime" already defined (by an import)
endpoints/api/test/test_security.py:32: error: Incompatible import of "org_view" (imported name has type "Callable[[Any], Any]", local name has type "Callable[[Any, Any], Any]")
endpoints/api/test/test_security.py:33: error: Name "datetime" already defined (by an import)
endpoints/api/test/test_security.py:37: error: Name "datetime" already defined (by an import)
```